templates/chef/resource.erb

```diff
@@ -124,6 +124,12 @@ module Google
 
 <% end # object.save_api_results? -%>
       action :create do
+<% if object.deprecated -%>
+        Chef.deprecated(:generic,
+                        ["<%= object.out_name -%> has been deprecated.",
+                         "Please use a string instead."].join(" "))
```

Comment:
Can you be more specific in this message?

Reply:
I mentioned this in the Puppet PR too.

What would be better to put in? I can't specify which value they should use, since it varies based on the ResourceRef.

Reply:
Is it the resource name, or the full URL?  Saying "you can specify the full URL as a string" or "You can specify the `<%= object.name -%>` name as a string" would work, right?